src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml

```diff
@@ -12,111 +12,108 @@ sylius_fixtures:
-                            en_GB: ~
-                            es_ES: ~
-                            pl_PL: ~
+                            %locale%: ~
 
                 currency:
                     options:
                         currencies:
-                            EUR: ~
-                            USD: ~
-                            GBP: ~
-                            CHF: ~
+                            %currency%: ~
 
                 geographical:
                     options:
-                        provinces:
-                            US:
-                                US-CA: California
-                                US-OR: Oregon
-                                US-WA: Washington
-                                US-AL: Alabama
-                                US-TN: Tennessee
+                        countries:
+                            - US
                         zones:
-                            US-WEST-COAST:
-                                name: "West coast of USA"
-                                provinces:
-                                    - US-CA
-                                    - US-OR
-                                    - US-WA
-                            EUROPE:
-                                name: "European countries"
+                            US:
+                                name: "United States of America"
                                 countries:
-                                    - PL
-                                    - DE
-                                    - FR
-                            YO-DAWG:
-                                name: "Zone in a zone"
-                                zones:
-                                    - US-WEST-COAST
-                                    - EUROPE
+                                    - US
 
                 payment_method:
                     options:
-                        random: 10
-
-                shipping_category:
-                    options:
-                        random: 10
+                        custom:
+                            cash_on_delivery:
+                                name: Cash on delivery
+                            bank_transfer:
+                                name: Bank transfer
 
                 shipping_method:
                     options:
-                        random: 20
+                        custom:
+                            ups:
+                                name: UPS
+                            dhl_express:
+                                name: DHL Express
+                            fedex:
+                                name: FedEx
 
                 channel:
                     options:
-                        random: 3
+                        custom:
+                            us_web_store:
+                                name: US Web Store
+                                locales:
+                                    - %locale%
```

Comment:
Newer versions of Symfony YAML needs this to be surrounded with quotation marks - `- "%locale%"`